tinygettext/PSPBUILD

```diff
@@ -0,0 +1,38 @@
+pkgname=tinygettext
+pkgver=20240715
+pkgrel=1
+pkgdesc="a simple gettext replacement that works directly on .po files"
+arch=('mips')
+license=('zlib')
+groups=('pspdev-default')
+url="https://github.com/tinygettext/tinygettext"
+depends=('sdl2')
+optdepends=('')
```

Comment:
Try to remove the empty string from optdepends and see if it builds. It doesn't reflect in the https://pspdev.github.io/psp-packages if it doesn't build all packages properly.

Reply:
Good catch, I fixed this now